docs/md_merge.py

```diff
@@ -0,0 +1,357 @@
+import argparse
+import os
+import re
+import shutil
+from argparse import BooleanOptionalAction
```

Comment:
Using `action=argparse. BooleanOptionalAction` down in line 332 should work nice and we could remove that special import line.

Reply:
updated